<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>a</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_26">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>c</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>b</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>d</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>f</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>j</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>e</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>h</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>i</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>g</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>q</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>n</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>m</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>l</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_15">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>k</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_16">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>o</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_17">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>p</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_18">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>r</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_19">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>t</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_20">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>u</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_21">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>v</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_22">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>w</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_23">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_24">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>y</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_25">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>z</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_alfabet</string>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>731</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="button_clear">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>s</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_probel">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string> </string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_clearback">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;backspace</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_alfabet"/>
 </buttongroups>
</ui>
